metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java

```diff
@@ -190,7 +196,22 @@ public void literal(final String name, final String value) {
         }
     }
 
-    private void processLiteralInLeader(final String name, final String value) {
+    private void processLeaderAsOneLiteral(final String value) {
+        if (value.length() != Iso2709Constants.RECORD_LABEL_LENGTH) {
+            throw new FormatException(
+                    "leader literal must contain " + Iso2709Constants.RECORD_LABEL_LENGTH + "  characters:" + value);
```

Comment:
Space after colon?

Reply:
done in eb38a1bd29cb7a306561248da428dd60f3c2eaae